client/js/chart/index.js

```diff
@@ -60,6 +132,24 @@ function createSeries(color, history) {
     start = end;
   }
 
+  // create marker for when the user mouses near a point
+  const marker = document.createElementNS(SVG_NS, 'circle');
+  marker.classList.add('series-marker');
+  marker.setAttribute('r', '5px');
```

Comment:
so there are many (hidden) points on the graph, and circle gets created when user hovers over it?

Reply:
The circle gets created at the beginning, but it is added to the DOM when the user hovers, and removed when they move their mouse away